src/contexts/Network/defaults.ts

```diff
@@ -11,6 +11,7 @@ import type {
 export const activeEra: ActiveEra = {
   index: new BigNumber(0),
   start: new BigNumber(0),
+  isPlaceholder: true,
```

Comment:
When does `isPlaceholder` become `false` ?

Reply:
Yeah I probably should have left a comment, it's a little tricksy.

The `isPlaceHolder` field is optional, and it's only (currently) set here to true. When we actually fetch the real data, we don't set the `isPlaceholder` field at all - so it's undefined. So if the `isPlaceholder` is true, then it's a placeholder. If `isPlaceholder` is `false` or `undefined` then it is not a placeholder.

Conveniently `if (activeEra.isPlaceholder)` behaves appropriately then

Reply:
The original intent was to modify as little of the code as possible, but now that I'm looking at it, it wouldn't be much more effort to just make `isPlaceholder` mandatory and either `true`/`false`.

I'll change it so it's more intuitive